[2023-12-20T11:58:24.334+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.second_id scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-12-20T11:58:24.375+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.second_id scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-12-20T11:58:24.376+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:58:24.622+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2022-04-19 00:00:00+00:00
[2023-12-20T11:58:24.641+0000] {standard_task_runner.py:60} INFO - Started process 9052 to run task
[2023-12-20T11:58:24.673+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'second_id', 'scheduled__2022-04-19T00:00:00+00:00', '--job-id', '2369', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmp427xnib9']
[2023-12-20T11:58:24.686+0000] {standard_task_runner.py:88} INFO - Job 2369: Subtask second_id
[2023-12-20T11:58:25.003+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v4.second_id scheduled__2022-04-19T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:58:25.403+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2022-04-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-19T00:00:00+00:00'
[2023-12-20T11:58:25.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:58:25.406+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 1']
[2023-12-20T11:58:25.460+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:58:25.462+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 1
[2023-12-20T11:58:25.464+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:58:25.555+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=second_id, execution_date=20220419T000000, start_date=20231220T115824, end_date=20231220T115825
[2023-12-20T11:58:25.622+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:58:25.707+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
